Client/metrics.yaml

```diff
@@ -325,6 +325,18 @@ tabs:
     notification_emails:
       - firefox-ios@mozilla.com
     expires: "2021-07-01"
+  add-new-tab-button:
```

Comment:
I don't see the update to the metrics.md or anything that suggests that this was processed by the glean_parser build step correctly.  If you look at the errors to the build, do you see any "Glean Parser" errors or warnings? If so, you need to fix those for the code generation step to work.  

Also, just a naming nit, but this means you will be calling this like `GleanMetrics.Tabs.addNewTabButton.add()`, which is a little awkward. What if you shortened it to `new-tab-button`?

Reply:
Is this different than the tab open counter above? I understood it to be the event that fired when a new tab is opened, but this is fine if there are multiple ways of opening a new tab and you this is just counting one of the ways.

Reply:
Yeah we have a new tab button at the tab toolbar and this one is on the url nav bar hence the difference.

Reply:
@travis79 Took your suggestion and renamed it to be `new_tab_pressed` :)